src/Routers/ClassesRouter.js

```diff
@@ -57,14 +57,14 @@ export class ClassesRouter extends PromiseRouter {
       }
     }
 
-    if (typeof body.keys === 'string') {
-      options.keys = body.keys;
+    if (body.keys || body.keys === '') {
```

Comment:
The `=== ''` is because we want to support `select([])`?

Reply:
Yup, that’s what makes it work with the update

Reply:
That's pretty ugly, but if there's no other way... :/

Reply:
This implementation probably has something to do with the JS SDK taking a delineated string, so it’s needed to pass the current tests. It can probably change once JS can take a string array

Reply:
How about `if (body.keys != null) {` ? That covers any value set for `keys`, array, string or empty string.

Reply:
I need to verify this still works encoded array. I'll ping back if it does

Reply:
This is the current update on the PR and it works, `if (body.keys != null) {`

Reply:
Doesn't work: `if (typeof body.keys === 'string') {`

Above closely resembles what's being fixed in this PR